tests/networkpolicy_test.go

```diff
@@ -0,0 +1,246 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1network "k8s.io/api/networking/v1"
+	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Networkpolicy", func() {
+
+	flag.Parse()
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	var vmia *v1.VirtualMachineInstance
+	var vmib *v1.VirtualMachineInstance
+	var vmic *v1.VirtualMachineInstance
+
+	tests.BeforeAll(func() {
+		tests.SkipIfUseFlannel(virtClient)
+		tests.BeforeTestCleanup()
```

Comment:
if you will put it under `BeforeAll` it will run only once, does it intentionally?

Reply:
Yes, it is intentional, since we just create different networkpolicy rules and don't make any change for the vmis.